packages/contracts/contracts/libraries/bridge/CrossDomainEnabled.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >0.5.0 <0.8.0;
+pragma solidity ^0.8.7;
```

Comment:
This being intended as a reusable lib should probably also have the more general version range: `pragma solidity >0.5.0 <0.9.0`